addons/web/static/src/js/core/smooth_scroll_on_drag.js

```diff
@@ -252,13 +250,13 @@ const SmoothScrollOnDrag = Class.extend(mixins.ParentedMixin, {
 
         // If this.$scrollTarget is the html tag, we need to take the scroll position in to account
         // as offsets positions are calculated relative to the document (thus <html>).
+        const scrollTargetScrollTop = this.$scrollTarget.scrollTop();
```

Comment:
Hello!
I don't understand the change you made. For me either you move all the code after the condition or you don't move any. And do not forget to change the comment above accordingly 😉 

The history behind those lines is that I had a different behavior in the web_editor and in sign. Those lines made it work everywhere although I did never really understood the reason behind.
Please be sure that you do not break anything.

cc: @tivisse @qsm-odoo

Reply:
The system before did not work on all dom structures. We have elements that drag and drop into a box where one (and sometimes more than one) parent has a scroll bar. And the source of the drag & drop being outside one of the scrolls (but in the second when there are several scrolls). It was similarly that did not work if the destination was in an iframe.